test/integration/test_navbar_links.js

```diff
@@ -0,0 +1,139 @@
+tap.test('checkCreateLinkWhenSignedOut', function (t) {
+    var xPathLink = '//li[contains(@class, "link") and contains(@class, "create")]/a';
+    var checkCreateLinkWhenSignedOut = driver.findElement(seleniumWebdriver.By.xpath(xPathLink));
+    checkCreateLinkWhenSignedOut.getAttribute('href').then( function (url) {
+        //expected value of the href
+        var expectedHref = '/projects/editor/?tip_bar=home';
+        //the create href should match `/projects/editor/?tip_bar=home`
+        //the create href should be at the end of the URL
+        t.equal(url.substr(-expectedHref.length), expectedHref);
+        t.end();
+    });
+});
+
+tap.test('checkExploreLinkWhenSignedOut', function (t) {
+    var xPathLink = '//li[contains(@class, "link") and contains(@class, "explore")]/a';
+    var checkExploreLinkWhenSignedOut = driver.findElement(seleniumWebdriver.By.xpath(xPathLink));
+    checkExploreLinkWhenSignedOut.getAttribute('href').then( function (url) {
+        //expected value of the href
+        var expectedHref = '/explore/projects/all';
+        t.equal(url.substr(-expectedHref.length), expectedHref);
+        t.end();
+    });
+});
+
+tap.test('checkDiscussLinkWhenSignedOut', function (t) {
+    var xPathLink = '//li[contains(@class, "link") and contains(@class, "discuss")]/a';
+    var checkDiscussLinkWhenSignedOut = driver.findElement(seleniumWebdriver.By.xpath(xPathLink));
+    checkDiscussLinkWhenSignedOut.getAttribute('href').then( function (url) {
+        //expected value of the href
+        var expectedHref = '/discuss';
+        t.equal(url.substr(-expectedHref.length), expectedHref);
+        t.end();
+    });
+});
+
+tap.test('checkAboutLinkWhenSignedOut', function (t) {
+    var xPathLink = '//li[contains(@class, "link") and contains(@class, "about")]/a';
+    var checkAboutLinkWhenSignedOut = driver.findElement(seleniumWebdriver.By.xpath(xPathLink));
+    checkAboutLinkWhenSignedOut.getAttribute('href').then( function (url) {
+        //expected value of the href
+        var expectedHref = '/about';
+        t.equal(url.substr(-expectedHref.length), expectedHref);
+        t.end();
+    });
+});
+
+tap.test('checkHelpLinkWhenSignedOut', function (t) {
+    var xPathLink = '//li[contains(@class, "link") and contains(@class, "help")]/a';
+    var checkHelpLinkWhenSignedOut = driver.findElement(seleniumWebdriver.By.xpath(xPathLink));
+    checkHelpLinkWhenSignedOut.getAttribute('href').then( function (url) {
+        //expected value of the href
+        var expectedHref = '/help';
+        t.equal(url.substr(-expectedHref.length), expectedHref);
+        t.end();
+    });
+});
+
+// ==== Search bar ====
+
+tap.test('checkSearchBar', function (t) {
+    var xPathLink = '//input[@id="frc-q-1088"]';
+    // search bar should exist
+    // if it isn't found, it will throw an error, which tap will treat as a failure
+    var checkSearchBar = driver.findElement(seleniumWebdriver.By.xpath(xPathLink));
```

Comment:
If possible, I think it would be better to be explicit about this and assert that there should be one element that matches the path, rather than allowing the raised error to implicitly fail the the test. This way if some other error is causing the failure, you can distinguish that from the element not being on the page.